rust/src/randomize.rs

```diff
@@ -2963,13 +2966,13 @@ impl<'r> Randomizer<'r> {
             }
         }
         info!(
-            "[attempt {attempt_num_rando}] Finishing with {:?}",
+            "[attempt {attempt_num_rando}] Finishing without {:?}",
             remaining_items
         );
         let mut idx = 0;
         for item_loc_state in &mut state.item_location_state {
             if item_loc_state.placed_item.is_none() {
-                item_loc_state.placed_item = Some(remaining_items[idx]);
+                item_loc_state.placed_item = Some(Item::Nothing);
```

Comment:
I think Nothing items should only be placed if the player opted into a reduced item pool in some way. On Basic, there will typically be an unreachable item in Main Street but we normally still want to put a filler item there.

Reply:
Everything else in the PR looks good. It lays a nice foundation for future stuff we'll want to do with Nothing items. For Escape-only mode, we could also switch it over to use Nothing items instead of Missiles: https://github.com/blkerby/MapRandomizer/blob/beffb5c45e3fb4f94a9d38a3b532447ca442e413/rust/src/randomize.rs#L3658.

Reply:
Fixed both. I was under the assumption that it would be nice to indicate that the item isn't supposed to be collectable, but that's also fair.